app/controllers/catalog_controller.rb

```diff
@@ -41,6 +42,31 @@ def update_user_per_cat_page
     end
   end
 
+  def submit_cart_updates_form # rubocop:disable MethodLength, AbcSize
+    flash.clear
+    begin
+      # set the start and end date to updated values
+      # converts the dates into the expected format
+      start_date = Date.strptime(params[:form][:start_date], '%m/%d/%Y')
```

Comment:
Alright, I'll go ahead and change it then.
2015年10月4日 下午12:04，"Oren Kanner" <notifications@github.com>写道：

> In app/controllers/catalog_controller.rb
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_YaleSTC_reservations_pull_1221-23discussion-5Fr41098984&d=AwMCaQ&c=-dg2m7zWuuDZ0MUcV7Sdqw&r=fGUlyXMXUllcq1Lzipj1Qfu_NL-lk9lC6GgNV_p2VOg&m=oAG60cY2eRqZe6-ZnhgdVfvTcnG9vRkRDdkZ-kfjGew&s=AtqsfckVLR490le6HF_O1Bc9QNb1U1wXsQS_D_Kf-lw&e=>
> :
>
> > @@ -41,6 +42,31 @@ def update_user_per_cat_page
> >      end
> >    end
> >
> > +  def submit_cart_updates_form # rubocop:disable MethodLength, AbcSize
> > +    flash.clear
> > +    begin
> > +      # set the start and end date to updated values
> > +      # converts the dates into the expected format
> > +      start_date = Date.strptime(params[:form][:start_date], '%m/%d/%Y')
>
> I hear that, but ideally we should just be consistent between the two
> forms, one way or another. Since we've already got working JS code to
> handle the datepickers and extracting the data for the cart updates, we can
> basically embed the same form (structurally) on this page, which will allow
> us to use the update_cart method unmodified.
>
> —
> Reply to this email directly or view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_YaleSTC_reservations_pull_1221_files-23r41098984&d=AwMCaQ&c=-dg2m7zWuuDZ0MUcV7Sdqw&r=fGUlyXMXUllcq1Lzipj1Qfu_NL-lk9lC6GgNV_p2VOg&m=oAG60cY2eRqZe6-ZnhgdVfvTcnG9vRkRDdkZ-kfjGew&s=fpM1YIFS-HEwjbXWLDPFoJ_lyHSW08MfpemDUQYbAOY&e=>
> .
>